packages/runtime/runtime-utils/src/runtimeFactoryHelper.ts

```diff
@@ -15,15 +15,10 @@ export abstract class RuntimeFactoryHelper<T = IContainerRuntime> implements IRu
 
     public async instantiateRuntime(
         context: IContainerContext,
-        existing?: boolean,
+        existing: boolean,
```

Comment:
This should be made required in the `IRuntimeFactory` interface as well. I see this happening in the container-definitions.api.md but not in the actual interface?